subprojects/docs/src/docs/release/notes.md

```diff
@@ -381,8 +381,13 @@ testLauncher.withTestsFor(spec -> {
 ```
 
 
+### IDE integration improvements
 
+#### Tooling API can control system properties passed to the build
```

Comment:
```suggestion
#### Added support for passing system properties to the build with the Tooling API
```
Suggestion: rephrase for consistency with other section titles in the release notes.

Reply:
Thank you for the help, @nathan-contino!